src/Api/OperationType.php

```diff
@@ -18,5 +18,6 @@ final class OperationType
     public const ITEM = 'item';
     public const COLLECTION = 'collection';
     public const SUBRESOURCE = 'subresource';
+    public const RESOURCE = 'resource';
```

Comment:
"Metadata" is even weirder IMO since the operation types are used to get specific properties in the resource metadata.
I see it more like a level of the metadata: at the `resource` level, at the `item` (operation) level.
Maybe it's not the best name, but I don't have another one.

Reply:
Hmm... Isn't it akin to the HTTP `HEAD` method?

Reply:
Hmm not really? It's mainly used by the `getTypedOperationAttribute` of the `ResourceMetadata` to retrieve the attributes of the resource, and not the attributes of a specific operation.

Reply:
Okay, then that's very hackish. An "operation type" that's not actually an operation type. :see_no_evil:

Reply:
Kind of, yeah... If you have a better idea, I'm taking it!

Reply:
It definitely must not be resource. It's misleading.

Reply:
I don't understand why it's misleading. We set the attributes at the *resource* level. I don't think we ever used another term to describe it.
We don't say at the "global" level. Or at the "metadata" level. Nor at the "root" level. Even it's not a perfect fit, because by essence it's not really an "operation" (and we can ask ourselves what a "subresource" operation is...), I don't think there is a better alternative.